app/src/main/java/com/sdp/movemeet/view/activity/ActivityDescriptionActivity.java

```diff
@@ -207,6 +213,7 @@ private void createAddressView() {
      * Syncing registered participant to Firebase Firestore (field array "participantId")
      */
     public void registerToActivity(View v) {
+        userId = fAuth.getCurrentUser().getUid();
```

Comment:
Why do you redefine ("re-get") the `userId` here? We already get it in the `onCreate`, aren't we?

Reply:
Indeed. This is probably duplicate code due to a merge with the main, which I didn't catch afterwards

Reply:
Ok alright, make sure to change this then (if you are able to) 😉.